concourse/util.py

```diff
@@ -90,3 +92,52 @@ def _enumerate_github_org_configs(job_mapping_set: JobMappingSet,):
 
         for github_org_config in github_org_configs:
             yield (github_org_config.org_name(), github_org_config.github_cfg_name())
+
+
+def resurrect_pods(
+    namespace: str,
+    label_selector: str,
+    concourse_client,
+    kubernetes_client,
+):
```

Comment:
add a few words as to what this function is all about? as in: concourse pods tend to crash and need to be pruned to help with the self-healing. no need to a lengthy essay, but a few hints might be good methinks